.github/workflows/pr-tests.yml

```diff
@@ -41,6 +41,10 @@ jobs:
       - name: Date of the restored cache
         run: cat target/cache-build-date.txt
         continue-on-error: true
+      - name: Install c++ dependencies for EStarkPolygon prover
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y uuid-dev libgrpc++-dev
```

Comment:
This block can be merged with the one below since they're both installing things for EStark